browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         },
         "14": {
           "release_date": "2012-06-26",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/14",
+          "release_notes": "https://www.mozilla.org/firefox/android/14.0beta/releasenotes/",
           "status": "retired"
         },
         "15": {
           "release_date": "2012-08-28",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/15",
+          "release_notes": "https://www.mozilla.org/firefox/android/15.0beta/releasenotes/",
           "status": "retired"
         },
         "16": {
           "release_date": "2012-10-09",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/16",
+          "release_notes": "https://www.mozilla.org/firefox/android/16.0beta/releasenotes/",
           "status": "retired"
         },
         "17": {
           "release_date": "2012-11-20",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/17",
+          "release_notes": "https://www.mozilla.org/firefox/android/17.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/17.0/releasenotes/).